app/models/support/requests/content_publisher_feedback_request.rb

```diff
@@ -0,0 +1,41 @@
+require 'active_support/core_ext'
+
+module Support
+  module Requests
+    class ContentPublisherFeedbackRequest < Request
+      attr_accessor :feedback_type, :feedback_details, :impact_on_work
+
+      OPTIONS = {
+        "accessibility or usability" => "accessibility or usability",
+        "working unexpectedly" => "something working differently than you expected",
+        "helpful feature" => "a feature that would help you with your work",
+        "other" => "something else",
+      }.freeze
+
+      validates_presence_of :feedback_type, :feedback_details
+      validates :feedback_type, inclusion: { in: OPTIONS.keys }
+
+
+
```

Comment:
Looks like we have some superfluous new lines to remove

Reply:
Done. Spaces removed, it's interesting to note how line spaces are done from request to request.